lib/functions.sh

```diff
@@ -116,7 +116,7 @@ assertVarsNotEmpty() {
 	MISSING_VARS=""
 
 	for VAR in $@; do
-	if [ -z "${!VAR}" ]; then
+		if [ -z "${VAR+x}" ] || [ -z "$VAR" ]; then
```

Comment:
What is the intention of this change?

Reply:
This seems to break the original intention of the code so I changed it to how it was before.